go.mod

```diff
@@ -1,27 +1,90 @@
 module github.com/codingben/kubevirt-disk-uploader
 
-go 1.22.4
+go 1.22.2
```

Comment:
please divide these change in commits e.g. like this:
1) code changes
2) go.mod, go.sum
3) vendor dependencies

Reply:
Yep, I'll do it before merging this PR.